analysis/analysis.py

```diff
@@ -1221,6 +1263,28 @@ def setUp(self):
         self.ads = "This is an ad."
         self.short_story = "This is actually a short story."
 
+    def test_unique_words_pattern(self):
+        """
+                Tests unique_words_pattern function with two data sets. The first is
```

Comment:
Please regularize indentation of this comment, so it's easier to read. (All indents at same level).

Reply:
Fixed